### DISCUSSION TITLE
Allow to create new instance with a different port behind reverse proxy

### DISCUSSION THREAD

**Discussion by JavaZeroo** [nextcloud/all-in-one#943] [Ideas] [answered]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments. 

### Steps to reproduce
1. Follow the [https://github.com/nextcloud/all-in-one/blob/main/reverse-proxy.md](Reverse Proxy Documentation)
2. login to New AIO instance page

- type my domain `***.top`  Nextcloud AIO will prompt "The server is not reachable on Port 443. You can verify this e.g. with 'https://portchecker.co/' by entering your domain there as ip-address and port 443 as port."
- type my doamin `***.top:8080`  Nextcloud AIO will prompt "Domain is not a valid domain!" 

 
### Expected behavior 
You can specify a port when creating a new instance
### Actual behavior 
When creating an instance, when you cannot use port 443, you cannot create an instance

### Host OS 
Ubuntu 22.24

#### Nextcloud AIO version 
Latest
#### Current channel 
Stable

**Comment by szaimen** [comment]
Mkving to discussions since thid is a feature request

**Comment by szaimen** [comment]
Why are you not able to use port 443 for AIO?

**Reply by JavaZeroo** [reply]
In China, in order to prevent people from using home broadband to build websites, ports 443 and 80 are blocked. :(

**Reply by szaimen** [reply]
I see. Is it maybe possible to user Cloudflare Argo Tunnel or is the whole Cloudflare network blocked in China?

**Reply by szaimen** [reply]
Maybe using the dns challenge would also be a possibility?
See https://github.com/nextcloud/all-in-one/issues/876

**Reply by JavaZeroo** [reply]
> Maybe using the dns challenge would also be a possibility? See #876

omg I think it is a better solution. THANKS, I will give it a try.

**Reply by szaimen** [reply]
Here is the documentation on this: https://github.com/nextcloud/all-in-one/tree/enh/876/dns-challenge#how-to-get-it-running-using-the-acme-dns-challenge

**Comment by szaimen** [comment, marked as answer]
See https://github.com/nextcloud/all-in-one/discussions/839#discussioncomment-2992521

**Reply by JavaZeroo** [reply]
thanks!! I will try to manual install it. =3=

**Reply by szaimen** [reply]
All right! I hope it works for you!
